packages/embark/src/lib/utils/solidity/remapImports.ts

```diff
@@ -10,8 +10,7 @@ const FIND_FILE_REGEX = /import[\s]*(['"])(.*)\1;/;
 
 interface RemapImport {
   path: string;
-  searchValue: string;
-  replaceValue: string;
+  remapping: ImportRemapping;
```

Comment:
Should import remapping be declared here?
It seems to be lower in the stack

Reply:
I'm confused, not sure what you mean?

Reply:
Oh ok. Updated.